tasks/run-matlab-build/v1/task.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
In this file, consider reusing the text for the MATLAB orb (CircleCI) or the MATLAB actions (GitHub Actions).

Reply:
Can we consider doing this in a separate PR before releasing? This matches the v0 help text from Azure so both the v0 and v1 should be updated at the same time.

Based on the CircleCI review, it seems it might take a few revisions, and this PR is already quite large,

Reply:
Sounds good.